providers/implementations/storemgmt/file_store.c

```diff
@@ -762,6 +764,9 @@ static int file_close_stream(struct file_ctx_st *ctx)
 
 static int file_close(void *loaderctx)
 {
+    if (loaderctx == NULL)
+        assert(0);
+    
```

Comment:
I do not think we want such assert calls. Please drop this.

Reply:
done